install/tools/ipactl

```diff
@@ -138,7 +138,8 @@ def version_check():
     try:
         installutils.check_version()
     except (installutils.UpgradeMissingVersionError,
-            installutils.UpgradeDataOlderVersionError):
+            installutils.UpgradeDataOlderVersionError) as exc:
+        emit_err("IPA version error: %s" % exc)
```

Comment:
I'm fine with just changing this particular line, I don't think that in this case is needed to keep a consistency.